### ISSUE TITLE
In Mozilla Firefox Browse Mode, NVDA repeats the level for every row in a tree grid

### ISSUE THREAD

**Issue by LeonarddeR** [nvaccess/nvda#16885]
### Steps to reproduce:
1. Open this:
`data:text/html, <table role="treegrid"><tr><td>hello 1</td></tr><tr><td>Hello 2</td></tr></table>`
2. Move to the table with up and down arrows.

### Actual behavior:
NVDA announces level 1 for every row

### Expected behavior:
NVDA should only announce the level when it changed compared to the previous level.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-32984 (2024.4 release cycle)

#### Windows version:
Windows 11 24H2

#### Name and version of other software in use when reproducing the issue:
Firefox 128

#### Other information about your system:
With Edge, the issue does not occur, though there the level seems to be missing from the outline items accessibility info anyway.

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by XLTechie** [comment]
Whereas in Chromium, no level is ever announced.

**Comment by Adriani90** [comment]
@LeonarddeR can you still reproduce this? I am testing with the last NVDA 64bit alpha in Firefox 145 or Chrome 144, and none of them report the level at all.
